maven/pom.xml

```diff
@@ -252,6 +247,33 @@
                     </lifecycleMappingMetadata>
                   </configuration>
                 </plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-dependency-plugin</artifactId>
+					<version>2.10</version>
+					<configuration>
```

Comment:
It is not clear to me that this configuration should be in the parent pom. It is shared by 2 of the plugins but if we need to use the dependency plugin in another plugin, that plugin may not need the embedded repo. This config should be in each project's pom.